bcda/cclf/cclf.go

```diff
@@ -300,6 +301,10 @@ func importCCLF(fileMetadata *cclfFileMetadata, importFunc func(uint, []byte, *g
 }
 
 func getCCLFFileMetadata(filePath string) (cclfFileMetadata, error) {
+	if strings.Contains(filePath,"BCD") {
+		return getCCLFFileMetadataBCD(filePath)
+	}
+
 	var metadata cclfFileMetadata
 	// CCLF filename convention for SSP: P.A****.ACO.ZC*Y**.Dyymmdd.Thhmmsst
```

Comment:
The `len(filenameMatches)` count below (line 308/313) should be fixed with this ticket. For the existing pattern, I believe it's 6 instead of 5.